doc/optimizers.md

```diff
@@ -465,7 +465,7 @@ approxOptimizer.Optimize(f, coordinates);
 
 *An optimizer for [arbitrary functions](#arbitrary-functions).*
 
-Conventional Neural Evolution is an optimizer that works like biological evolution which selects best candidates based on their fitness scores and creates new generation by mutation and crossover of population. The initial population is generated based on a normal distibution centered at the given initial point.
+Conventional Neural Evolution is an optimizer that works like biological evolution which selects best candidates based on their fitness scores and creates new generation by mutation and crossover of population. The initial population is generated based on a random normal distibution centered at the given initial point.
```

Comment:
Should we use "a population" and "distribution" instead of "distibution"?

Reply:
Do you mean "a population" instead of "initial population"? If so, i think "initial population" is more descriptive, since subsequent populations are generated by crossover and mutation.

Reply:
Then again, I suppose you can look at it as new generations being created. not new populations. Seems to be an issue of semantics then, I'll go with your judgement.